pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -44,6 +44,7 @@
     'andor': lambda x, y, z: x and y or z
 }
 
+DEBUG_ON = False
 
```

Comment:
It's not unusual. A lot of other code has the same thing. It's actually really handy when debugging a single module.